template.ego

```diff
@@ -41,6 +41,13 @@ var NotFound = http.NotFound
 
 // ServeHTTP serves a request, attempting to reply with an embedded file.
 func ServeHTTP(rw http.ResponseWriter, req *http.Request) {
+  if req.URL.Path == "/" {
```

Comment:
What if I have `/subdir/index.html`? There's probably a way to make this generic